src/common/ceph_context.cc

```diff
@@ -771,7 +771,7 @@ CephContext::CephContext(uint32_t module_type_,
   _admin_socket->register_command(
       "config diff get name=var,type=CephString", _admin_hook,
       "dump diff get <field>: dump diff of current and default config setting <field>");
-  _admin_socket->register_command("injectargs name=injected_args,type=CephString,n=N", _admin_hook, "inject configuration arguments into running daemon"),
+  _admin_socket->register_command("injectargs name=injected_args,type=CephString,n=N", _admin_hook, "inject configuration arguments into running daemon");
```

Comment:
thank you for the change. and i am for it. but please add more details on the impact of the existing two expression-single statement arrangement, and what your fix changes. presumably, it is just a cleanup to be more consistent. but still, please put the rationales in the commit message. "wrong delimiter" is not good enough.